packages/apps/marketplace/src/pages/Marketplace/index.tsx

```diff
@@ -127,11 +127,13 @@ const Marketplace = () => {
     let intervalId: any;
     if (actor) {
       fetchData();
-      if (!intervalId) {
+      if (!intervalId && inputText != '') {
```

Comment:
Looks like you want to prevent the refresh while search is active. We want to see updates in the filtered items as well. Please find a solution where the data can refresh in the background every 5 seconds while data is filtered.